README.md

```diff
@@ -1 +1,61 @@
-# hello-microverse
\ No newline at end of file
+# Hello Microverse
+> Setting up first "hello miceroverse" project.
+
+![image](https://user-images.githubusercontent.com/12495712/148043597-c6a72abe-64a9-4e2b-b776-6393348fac79.png)
+
+This project objective was to set up "hello microverse" project 
+
+## Built With
+
+- HTML and CSS
+- VSCode, GitHub
+
+## Live Demo
+
+[Live Demo Link](https://diwakarstha.github.io/hello-microverse/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
```

Comment:
- [ ] [OPTIONAL] I kindly suggest you to delete all the unnecessary lines from your `README` 😉

Reply:
I'm sorry I thought I had to create another pull request to submit again and I have made the changes by removing those lines and have also added MIT.md file.